extensions/amp-bind/0.1/bind-expr-impl.js

```diff
@@ -209,7 +209,7 @@ case 26:
 break;
 case 27:
 
-        this.$ = new AstNode(AstNodeType.INVOCATION, [null, $$[$0]], $$[$0-1])
+        this.$ = new AstNode(AstNodeType.INVOCATION, [undefined, $$[$0]], $$[$0-1])
```

Comment:
To differentiate between the lack of a caller (e.g. Math.abs) and a caller that doesn't exist (null). This change mirrors the one in the jison file.